### DISCUSSION TITLE
(Beginner) Importing never_split_tokens

### DISCUSSION THREAD

**Discussion by YomnaEid** [aub-mind/arabert#189] [Q&A] [answered]
When I run this code:
**from arabert.preprocess import never_split_tokens**
I got this error:
_Cannot import name 'never_split_tokens' from 'arabert.preprocess' (/usr/local/lib/python3.8/dist-packages/arabert/preprocess.py)_

So how to import it?

**Comment by WissamAntoun** [comment, marked as answer]
this is because you are using a very old version of the training scripts , this has now been updated to 
```python
from arabert.preprocess import NEVER_SPLIT_TOKENS
```

You also shouldn't need to import any of these tokens if you are doing fine-tuning with arabert
